<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditContact</class>
 <widget class="QDialog" name="EditContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	border: none;
	background-color: white;
	padding: 0;
	margin: 0;
	color: black;
}

#title {
	font-size: 28px;
	font-weight: bold;
}

QLineEdit{
	border: 1px solid #C4C4C4;
	background-color: transparent;
	border-radius: 6px;
	padding: 4px;
}

QLineEdit:focus {
	border-color: #1C2028;
}

QWidget {
	background-color: #F9F9F9;
}

QPushButton {
	background-color: #007BFF; 
	color: white;
	padding: 8px 20px;
	border-radius: 4px;
	text-align: center;
	text-decoration: none;
}

QPushButton:hover {
	background-color: #42A5F5;
}

#btnCancel_2 {
	background: transparent;
	color: black;
	border: 1px solid #007BFF;
	text-decoration: none;
	padding-top: 7px;
	padding-bottom: 7px;
}

#btnCancel_2:hover {
	background-color: #F5F5F5;
}

QListWidget {
	background-color: #ffffff;
	border: 1px solid #d0d0d0;
	border-radius: 4px;
	font-size: 12px;
}

QListWidget::item {
    border-radius: 4px;
}
QListWidget::item:alternate {
    background-color: #f9f9f9;
}

QListWidget::item:hover {
    background-color: #f0f7ff;
    color: #202020;
}
QListWidget::item:selected {
    background-color: #e7f0fd;
    color: #202020;
    border: 1px solid #a0c4ff;
    border-radius: 4px;
}

QListWidget::item:selected:hover {
    background-color: #d7e5f7;
    color: #202020;
    border: 1px solid #7aaaff;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QLabel" name="title">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Edytuj kontakt</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="main" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>24</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="verticalSpacing">
           <number>24</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>ImiÄ™</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="inputFirstName_2"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Nazwisko</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="inputLastName_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>20</number>
            </property>
            <item>
             <layout class="QFormLayout" name="formLayout_7">
              <property name="horizontalSpacing">
               <number>30</number>
              </property>
              <property name="verticalSpacing">
               <number>6</number>
              </property>
              <item row="0" column="1">
               <widget class="QLineEdit" name="inputAddressStreet_2"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Ulica</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_8">
              <property name="horizontalSpacing">
               <number>6</number>
              </property>
              <item row="0" column="1">
               <widget class="QLineEdit" name="inputAddressNumber_2"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Numer domu</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>20</number>
            </property>
            <item>
             <layout class="QFormLayout" name="formLayout_9">
              <property name="horizontalSpacing">
               <number>20</number>
              </property>
              <item row="0" column="1">
               <widget class="QLineEdit" name="inputAddressCity_2"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Miasto</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_10">
              <item row="0" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Kod pocztowy</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="inputAddressPostalCode_2"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLineEdit" name="inputEmail">
              <property name="placeholderText">
               <string>Email</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnAddEmail">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::ListAdd"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="emailsList">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLineEdit" name="inputPhone">
              <property name="placeholderText">
               <string>Numer telefonu</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnAddPhone">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::ListAdd"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="phonesList">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="footer">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel_2">
        <property name="text">
         <string>Anuluj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Zapisz</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
